src/sprites.rs

```diff
@@ -10,6 +10,32 @@ extern crate ggez;
 use ggez::{GameResult, Context};
 use ggez::graphics::{self, set_color, Color, DrawMode};
 
+pub struct Crab_Sprite {
```

Comment:
The linter/compiler suggests that structs should use camelcase, so maybe change this to `CrabSprite` Otherwise, this looks great!

Reply:
Good catch